rest-api/tools/import_organizations.py

```diff
@@ -217,6 +222,13 @@ def _populate_lat_lng_and_time_zone(self, site, existing_site):
         site.longitude = longitude
         if latitude and longitude:
           site.timeZoneId = self._get_time_zone(latitude, longitude)
+    else:
+      if site.googleGroup not in self.status_exception_list:
```

Comment:
We also need a warning if the site *had* an address, but it wasn't geocodable.

Reply:
done